### DISCUSSION TITLE
x_frame_enabled = True is not work, X-Frame-Options still is DENY

### DISCUSSION THREAD

**Discussion by sjf159026** [apache/airflow#21841] [General]
### Apache Airflow version

2.2.3 (latest released)

### What happened

when display ariflow in another site in a iframe, the X-Frame-Options is deny, and x_frame_enabled  already set to True

### Operating System

CentOS Linux release 7.2.1511 (Core)

### Deployment

Docker-Compose

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
The issue description is completely useless. Please describe in detai what you tried, what did not work and generally more details of what you try to do, otherwise there is no way you can get any help

**Reply by sjf159026** [reply]
I just want to open airflow in a iframe  like <iframe src="http://jxxxx.xxx.se:10081/"> but the response is X-Frame-Options: Deny,
so can't open it, and I have set the config x_frame_enabled = True, so any idea about this?

**Comment by jedcunningham** [comment]
There will be a fix coming in Airflow 2.2.5: https://github.com/apache/airflow/pull/19491
